src/System.Private.CoreLib/shared/System/Type.Helpers.cs

```diff
@@ -263,7 +263,7 @@ public virtual MemberInfo[] FindMembers(MemberTypes memberType, BindingFlags bin
             }
 
             // Allocate the Member Info
-            MemberInfo[] ret = new MemberInfo[cnt];
+            MemberInfo?[] ret = new MemberInfo[cnt];
```

Comment:
This will not contain null values, same as the previous API. The return type shouldn't contain null values either. 

Same as: https://github.com/dotnet/coreclr/pull/23489/files/#r269756279

Reply:
yeah updating similarly